aten/src/ATen/core/dispatch/Dispatcher.h

```diff
@@ -338,6 +339,19 @@ class TORCH_API OperatorHandle {
     return operatorDef_->op.checkInvariants();
   }
 
+  const std::vector<at::Tag>& getTags() const {
+    return operatorDef_->op.getTags();
```

Comment:
Consider returning an ArrayRef instead, it makes less commitments about the internal representation and is harder to accidentally copy